packages/@react-spectrum/textfield/src/TextFieldBase.tsx

```diff
@@ -39,8 +37,6 @@ interface TextFieldBaseProps extends Omit<SpectrumTextFieldProps, 'onChange'>, P
 }
 
 function TextFieldBase(props: TextFieldBaseProps, ref: Ref<TextFieldRef>) {
-  props = useProviderProps(props);
```

Comment:
why was useProviderProps removed here but not in SearchWithin? or should it be the other way around?

Reply:
In my exploration I found having useProviderProps where did nothing. And it was needed in SearchWithin for the props like isQuiet, isRequired, isDisabled, etc. It is needed in any components that inherit those from the Provider. I was annoyed that I couldn't move useProviderProps to Field too.